.github/workflows/build_and_test.yml

```diff
@@ -4378,6 +4442,42 @@ jobs:
         call setenv.cmd
         cd tests/cmake_integration/
         perl run_ci_tests.pl --no-shared --build-config=Debug
+    - name: build messenger tests
```

Comment:
Should we make a separate mwc file for this subset of tests to match what's in the new test list? We can re-call mwc after the configure step if we don't feel like changing configure just for the static builds.

Reply:
mwc file would be found recursively so I'm hesitant to create a new mwc file rather than just manually running it.